pkg/kcp/provider/gcp/nfsinstance/syncNfsInstance.go

```diff
@@ -23,7 +23,7 @@ func syncNfsInstance(ctx context.Context, st composed.State) (error, context.Con
 	gcpScope := state.Scope().Spec.Scope.Gcp
 	project := gcpScope.Project
 	location := state.getGcpLocation()
-	name := nfsInstance.Spec.RemoteRef.Name
+	name := fmt.Sprintf("cm-%.60s", nfsInstance.Name)
```

Comment:
I don't think there is a need to use `%.60s` to truncate to 60 characters.
UUIDs are 36 characters long, so in our current use case, no truncation will happen.

If in the future, we decide to change UUIDs to something that can potentially be over 60 characters, then truncating might introduce a bug where due to truncation, two different NFS volumes get truncated into the same name (as they have the same 60char prefix).
For that (unlikely) scenario, In my opinion, it is better to let google API return an error, than to load a wrong NFS due to naming collision

What do you think @ravi-shankar-sap ?